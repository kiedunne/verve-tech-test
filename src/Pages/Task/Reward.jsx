import React from 'react';
import PropTypes from 'prop-types';
import './Reward.scss'

const Reward = ({reward}) => {
const { description, image } = reward
   return (
    <div className="Reward">
        <p>{description}</p>
        <img src={image}/>

    </div>
   )
  
}


export default Reward;
