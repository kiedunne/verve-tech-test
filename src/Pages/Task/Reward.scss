.Reward {
    display: flex;
    flex-direction: column;
    align-content: center;
    justify-content: center;
    width: 45%;
    padding: 2vw;
    margin: 1vw;
    background-color: #EBE1B7;
    border-radius: 30px;
    font-weight: bold;
    font-family: sans-serif;
    color: white;
}

