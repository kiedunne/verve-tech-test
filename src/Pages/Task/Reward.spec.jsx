import React from 'react';
import { shallow } from 'enzyme';
import  Reward  from './Reward';

describe('<Reward />', () => {
    let component;
    const props = {
        reward: [
            {
                image: 'https://google.com',
                description: 'a reward',
            }
        ],
    };

    beforeEach(() => {
        component = shallow(<Reward {...props} />);
    });

    it('renders with description and image props', () => {
        expect(component).toMatchSnapshot();
    });
});
