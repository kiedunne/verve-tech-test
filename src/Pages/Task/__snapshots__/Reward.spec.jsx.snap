// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Reward /> renders with description and image props 1`] = `
<div
  className="Reward"
>
  <p />
  <img />
</div>
`;
